pkg/bpf/bpffs.go

```diff
@@ -162,7 +164,10 @@ func mountFS() error {
 
 	mountMutex.Lock()
 	for _, m := range delayedOpens {
-		m.OpenOrCreate()
+		_, err = m.OpenOrCreate()
+		if err != nil {
+			log.WithError(err).WithField(logfields.BPFMapName, m.name).Error("error opening map after bpffs was mounted")
```

Comment:
Should we fatal? If not, what is the point of logging an error if the user can't do anything about it. the `moutFS()` will never be called again